.travis/install.sh

```diff
@@ -33,17 +33,18 @@ py27)
   conda create -q -n test-environment python=2.7 pip \
     numpy nose coveralls flake8 pyyaml gsl fftw cmake swig scipy \
     compilers pkg-config setuptools_scm pytest pandas pytest-cov \
-    cython "camb>=1" isitgr traitlets mkauthlist
+    cython "camb>=1" isitgr traitlets
   ;;
 py36)
   conda create -q -n test-environment python=3.6 pip \
     numpy nose coveralls flake8 pyyaml gsl fftw cmake swig  scipy \
     compilers pkg-config setuptools_scm pytest pandas pytest-cov \
-    cython "camb>=1" isitgr traitlets mkauthlist
+    cython "camb>=1" isitgr traitlets
   ;;
 esac;
 
 # we have to activate the cond env before we install this
 source activate test-environment
 pip install https://github.com/JoeMcEwen/FAST-PT/archive/v3.0.2.tar.gz --no-deps
+pip install mkauthlist
```

Comment:
you want --no-deps here and then to make sure any deps are in the list of conda installs above